DataFormats/EgammaCandidates/src/classes_def.xml

```diff
@@ -239,6 +239,13 @@
    <version ClassVersion="3" checksum="3868678681"/>
   </class>
 
+  <!-- ValueMap<reco::GsfTrack> -->
+  <class name="edm::ValueMap<reco::GsfTrackRefVector>" />
+  <class name="edm::Wrapper<edm::ValueMap<reco::GsfTrackRefVector> >" />
+  <class name="std::vector<edm::RefVector<std::vector<reco::GsfTrack>,reco::GsfTrack,edm::refhelper::FindUsingAdvance<std::vector<reco::GsfTrack>,reco::GsfTrack> > >"/>
+  <class name="edm::Wrapper<edm::RefVector<std::vector<reco::GsfTrack>,reco::GsfTrack,edm::refhelper::FindUsingAdvance<std::vector<reco::GsfTrack>,reco::GsfTrack> > >"/>
+  <class name="std::vector<edm::Ref<std::vector<reco::GsfTrack>,reco::GsfTrack,edm::refhelper::FindUsingAdvance<std::vector<reco::GsfTrack>,reco::GsfTrack> > >"/>
+
```

Comment:
@winterchristian , normally `GsfTrackReco` type dictionaries should go in to `DataFormats/GsfTrackReco`. Any specific reason to add these dict in `DataFormats/EgammaCandidates` package

Reply:
@cms-sw/reconstruction-l2  any objections on moving these to `DataFormats/GsfTrackReco` ?

Reply:
No objection.  Sorry we missed it.